library/core/src/main/java/com/google/android/exoplayer2/source/MediaSourceEventListener.java

```diff
@@ -66,6 +67,13 @@ default void onLoadCompleted(
       LoadEventInfo loadEventInfo,
       MediaLoadData mediaLoadData) {}
 
+  default void onLoadCompleted(
```

Comment:
onLoadCompleted is about loading a resource, probably not a good idea to add metadata here.

Reply:
I see that for `MediaSourceEventListener:loadCompleted` listener it will also return start time, end time, type, etc., which I think are also metadata, so I put PRFT here as well.

Reply:
Now the event is moved from `onLoadCompleted` to `onPrftParsed` which is triggered when a PRFT box is decoded.